.github/workflows/reusable-e2e.yml

```diff
@@ -0,0 +1,328 @@
+on:
+  workflow_call:
```

Comment:
"workflow_call" declares this file to be a reusable workflow. Other workflows can **call** this file to run e2e files as part of their steps.